workers/Dockerfile.base

```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:16.04
+
+RUN \
+  apt-get update -qq && \
+  apt-get install -y \
+                     lsb-release && \
+  echo "deb http://archive.ubuntu.com/ubuntu $(lsb_release -sc) multiverse\n" \
+      >> /etc/apt/sources.list.d/added_repos.list && \
+  echo "deb http://cran.cnr.berkeley.edu/bin/linux/ubuntu $(lsb_release -sc)/" \
+      >> /etc/apt/sources.list.d/added_repos.list && \
+  apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E084DAB9 && \
+  apt-get update -qq && \
+  apt-get install -y \
+                     ed \
+                     git \
+		     mercurial \
+		     libcairo-dev \
+		     libedit-dev \
+                     lsb-release \
+		     python3 \
+		     python3-pip \
+		     r-base \
+		     r-base-dev \
+                     libxml2-dev \
+                     libcurl4-gnutls-dev \
+                     libssl-dev \
+                     curl \
+		     wget &&\
+  rm -rf /var/lib/apt/lists/*
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
+WORKDIR /home/user
+
+ENV R_LIBS "/usr/local/lib/R/site-library"
+
+COPY workers/r_dependencies.R .
+
+RUN Rscript r_dependencies.R
+
+COPY workers/requirements.txt .
+
+RUN \
+  pip3 --no-cache-dir install pip --upgrade && \
+  pip3 --no-cache-dir install setuptools --upgrade && \
+  pip3 --no-cache-dir install wheel --upgrade && \
+  pip3 --no-cache-dir install -r requirements.txt --upgrade && \
+  rm -rf /root/.cache
```

Comment:
Works for me. I was guessing you had done this iteratively: added the `rm` at line 49, then discovered the `--no-cache-dir` param but didn't remove line 49.